src/lib/loaders/loaders_without_authentication/vortex.ts

```diff
@@ -6,25 +6,17 @@ interface LoaderArgs {
 }
 
 export default (opts) => {
-  const { vortexLoaderWithAuthenticationFactory } = factories(opts)
```

Comment:
This and the `setup` was the most confusing part. Why was there a factory named 'with authentication' being used here (as this file is namespaced to be 'without authentication'), and why was there a `setup` that didn't really do anything?